Tasks/DownloadBuildArtifactsV0/main.ts

```diff
@@ -276,6 +276,9 @@ async function main(): Promise<void> {
 
                 if (artifact.resource.type.toLowerCase() === "container") {
                     var handler = new webHandlers.PersonalAccessTokenCredentialHandler(accessToken);
+                    // this variable uses to force enable zip download option, it is used only in test purpose and shouldn't be used for other reasons
+                    const forceEnableZipDownloadOption = tl.getVariable("FORCE_ENABLE_ZIP_DOWNLOAD");
```

Comment:
```suggestion
                    const forceEnableZipDownloadOption = tl.getVariable("SYSTEM.ForceEnableZipDownload");
```

There is a `SYSTEM.DisableZipDownload` variable used in this task, so I think it makes sense to have this variable name in the same style.
Also I think `FORCE_ENABLE_ZIP_DOWNLOAD` is generic enough that there is a possibility of somebody having this variable enabled for a different reason, e.g. for some external tool, which could cause a regression.

Reply:
Regarding the more specific name, I agree, I will change the name of this variable to avoid overlap with other variables defined in client environments.
Regarding this format of the variable SYSTEM.ForceEnableZipDownload, I am not sure that we should use a system prefix for it, in my understanding variables marked by system prefix should be set by TFS and in theory, it is possible the variable with the same name could be introduced on TFS side, in this way our variable that is used for testing purpose will be overlapped.